system-tests/tests/smoke/cre/environment-one-don-single-chain.toml

```diff
@@ -47,7 +47,7 @@
   [[nodesets.node_specs]]
     [nodesets.node_specs.node]
       docker_ctx = "../../../.."
-      docker_file = "plugins/chainlink.Dockerfile"
+      docker_file = "core/chainlink.Dockerfile"
```

Comment:
To confirm, all of these tests should now use the core/chainlink.Dockerfile as all "released" (non-experimental) plugins will be included in here. Please let me know if this is not correct.